pylintrc

```diff
@@ -42,9 +42,10 @@ file-header=#\n# Copyright \(c\) (2019|2020|2021), Neptune Labs Sp\. z o\.o\.\n#
 # W0401, W0614 Allow wildcard imports.
 # W0511 Allow TODO/FIXME comments.
 # W0703 Allow too broad except clause (Exception).
+# W1202 Allow logging-format-interpolation.
```

Comment:
Allow f-strings in logs formatter.
PyLint requirement for '%s' has to be some kind of joke...

More about it: https://stackoverflow.com/a/59027338/1565454